Graphics Algos/README.md

```diff
@@ -0,0 +1,13 @@
+## GRAPHICS ALGORITHMS
+
+This is a folder which contains graphics algorithms and their implementations.
+
+### Robo.c
+
+The robo.c program is a program which helps you in drawing a robot.
+The robot is drawn around a reference point. The DDA Line Algorithm was used in making the program.
+The DDA Line algorithm exists inside the program in a function so you can have a look at what the DDA line algorithm is,
+inside this program itself.<br><br>
```

Comment:
what's this <br><br> ?

Reply:
Just to shift the control to the next line. For appearance sake

Reply:
didn't want everything to be in one para itself. looked too clumsy

Reply:
I can remove the tags if you want though @prateekiiest

Reply:
ok, that's ok then.

I am merging this 😄